arch/risc-v/src/mpfs/Kconfig

```diff
@@ -285,6 +285,11 @@ config MPFS_I2C1
 	select ARCH_HAVE_I2CRESET
 	default n
 
+config MPFS_I2C2
```

Comment:
Just an idea; we have the PWM block called "COREPWM" indicating it being an FPGA IP; should we then call this as MPFS_COREI2C or such? This is fine I quess as well, just that how to configure it if you would like to add 2,3,4... more i2c blocks? Would this be possible with just configuration?

Reply:
Yes, sure, that makes more sense. Just didn't think about that stuff at all.

Reply:
I just realized after starting CoreSPI, that I call the (new) driver CoreSPI and the Kconfig is also CONFIG_MPFS_CORESPI.

So this makes sense to me as well.

Reply:
MPFS_COREI2C0 ? (I think the FPGA driver may have up to 16 channels, maybe start counting from 0)

Reply:
My naive thinking was just

config MPFS_CORESPI
	bool "CoreSPI FPGA IP block configured"
	default n
	depends on MPFS_HAVE_CORESPI

config MPFS_CORESPI_INSTANCES
	int "Amount of CoreSPI instances"
	default 1
	range 1 8
	depends on MPFS_CORESPI

Have not verified if this approach will work or no